assets/post-panel/components/CivilNavBarButtons.tsx

```diff
@@ -65,26 +65,30 @@ export class CivilNavBarButtons extends React.Component<CivilNavBarButtonsProps>
           <CivilLogo />
         </CivilButton>
         <IconSection>
-          <PanelButtonSign
-            open={this.props.isOpen! && this.props.openTab === 0}
-            onClick={ev => this.onClickIcon(ev, 0)}
-          >
-            <ArticleSignIcon color={this.props.isOpen! && this.props.openTab === 0 ? "#ffffff" : "#444444"} />
-            <CircleIndicator
-              border={!this.props.isOpen! || this.props.openTab !== 0}
-              indicatorColor={this.signIndicatorColor()}
-            />
-          </PanelButtonSign>
-          <PanelButtonPublish
-            open={this.props.isOpen! && this.props.openTab === 1}
-            onClick={ev => this.onClickIcon(ev, 1)}
-          >
-            <ArticleIndexIcon color={this.props.isOpen! && this.props.openTab === 1 ? "#ffffff" : "#444444"} />
-            <CircleIndicator
-              border={!this.props.isOpen! || this.props.openTab !== 1}
-              indicatorColor={this.indexIndicatorColor()}
-            />
-          </PanelButtonPublish>
+          <ToolTip explainerText="Sign" positionBottom={true} verticalOffset={10} width={50}>
+            <PanelButtonSign
+              open={this.props.isOpen! && this.props.openTab === 0}
+              onClick={ev => this.onClickIcon(ev, 0)}
+            >
+              <ArticleSignIcon color={this.props.isOpen! && this.props.openTab === 0 ? "#ffffff" : "#444444"} />
```

Comment:
should this be a theme value

Reply:
Didn't edit this line, just indented cause of being wrapped in tooltip now. That being said - doesn't need to be theme-controllable since this is WP-specific, but I moved the colors into our post panel styles constants.